README.md

```diff
@@ -194,7 +194,8 @@ func main() {
 	docs.SwaggerInfo.Version = "1.0"
 	docs.SwaggerInfo.Host = "petstore.swagger.io"
 	docs.SwaggerInfo.BasePath = "/v2"
-
+    docs.SwaggerInfo.Schemes = []string{"http", "https"}
```

Comment:
please align it.

Reply:
Sorry about alignment are u using it seems that my editor is replacing tabs with different number of spaces.